app/connection/robust-websocket.ts

```diff
@@ -187,58 +169,49 @@
                     );
                 }, opts.timeout);
                 ['open', 'close', 'message', 'error'].forEach(function (stdEvent) {
-                    realWs.addEventListener(stdEvent, function (event) {
+                    (realWs as any).addEventListener(stdEvent, function (event: any) {
                         self.dispatchEvent(event);
-
                         const cb = self[`on${stdEvent}`];
                         if (typeof cb === 'function') {
                             return cb.apply(self, arguments);
                         }
                     });
                 });
-
                 if (!opts.ignoreConnectivityEvents) {
                     attachConnectivityEvents();
                 }
             }
-
             if (opts.automaticOpen) {
                 newWebSocket();
             }
         };
-
-        RobustWebSocket.defaultOptions = {
+        (RobustWebSocket as any).defaultOptions = {
             // the time to wait before a successful connection
             // before the attempt is considered to have timed out
             timeout: 4000,
             // Given a CloseEvent or OnlineEvent and the RobustWebSocket state,
             // should a reconnect be attempted? Return the number of milliseconds to wait
             // to reconnect (or null or undefined to not), rather than true or false
-            shouldReconnect(event, ws) {
+            shouldReconnect(event: any, ws: any) {
                 if (event.code === 1008 || event.code === 1011) return;
                 return [0, 3000, 10000][ws.attempts];
             },
-
             // Flag to control whether attachement to navigator online/offline events
             // should be disabled.
             ignoreConnectivityEvents: false,
-
             // Create and connect the WebSocket when the instance is instantiated.
             // Defaults to true to match standard WebSocket behavior
             automaticOpen: true,
         };
-
         RobustWebSocket.prototype.binaryType = 'blob';
-
         // Taken from MDN https://developer.mozilla.org/en-US/docs/Web/API/EventTarget
-        RobustWebSocket.prototype.addEventListener = function (type, callback) {
+        RobustWebSocket.prototype.addEventListener = function (type: any, callback: any) {
```

Comment:
```suggestion
        RobustWebSocket.prototype.addEventListener = function (type: string, callback: Listener) {
```

Reply:
robust-websocket is a library. I don't think we should make any changes there. 
`ts-migrate` migrator appended the types in it.
I have now removed this file and added it as an npm package from https://www.npmjs.com/package/robust-websocket